src/containers/booklist.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable react/jsx-key */
```

Comment:
- [ ] Kindly ensure that you don't overuse the linter disable. Linters are meant to enforce code quality and too much linter disable means that there is something wrong.